packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -0,0 +1,297 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/scheduler.dart';
+
+import 'events.dart';
+import 'pointer_router.dart';
+
+class _MouseDetails {
+  _MouseDetails({this.sourceTimeStamp, this.globalPosition});
+
+  /// Recorded timestamp of the source pointer event that last reported its
+  /// position. May be null if the mouse pointer is removed while the mouse
+  /// pointer is being tracked.
+  final Duration sourceTimeStamp;
+
+  /// The pointer's global position when it triggered this event in logical
+  /// pixels. May be null if the mouse pointer is removed while the mouse
+  /// pointer is being tracked.
```

Comment:
here and above: first paragraph should be short so it looks good in dartdocs; split the paragraph into two at the sentence break.